tests/src/Kernel/BackwardCompatibility/FeaturedMediaPatternTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\Kernel\BackwardCompatibility;
+
+use Drupal\Tests\oe_bootstrap_theme\Kernel\AbstractKernelTestBase;
+use Drupal\Tests\oe_bootstrap_theme\Traits\BackwardCompatibilityTrait;
+use Symfony\Component\DomCrawler\Crawler;
+
+/**
+ * Tests backward compatibility for the featured media pattern.
+ */
+class FeaturedMediaPatternTest extends AbstractKernelTestBase {
```

Comment:
I presume that this entire test was made for the backward compatibility setting to be toggled and check the rendering, please remove, that can be done easly from the markuprendering test

Reply:
It was created because before the bc settings were not provided for patterns page... now i change the markup rendering method and it passing them, and moved them to featured media.